data/json/vehicles/carts.json

```diff
@@ -255,6 +279,31 @@
     ],
     "items": [ { "x": 0, "y": 0, "chance": 30, "items": [ "goggles_welding" ] } ]
   },
+  {
+    "id": "motordriven_welding_cart",
+    "type": "vehicle",
+    "name": "Motor Driven Welding Cart",
+    "blueprint": [ "&" ],
+    "parts": [
+      {
+        "x": 0,
+        "y": 0,
+        "part": [
+          "frame_cover",
```

Comment:
There's no welder in your welding cart.

Reply:
Well, I've re done this about 5 times now, can't believe I forgot that.